qbosdk/apis/preferences.py

```diff
@@ -15,4 +15,4 @@ def get(self):
         Returns:
             Dict in Preferences schema.
         """
-        return self._get_request('Preferences', Preferences.GET_PREFERENCES)
+        return self._get_request(Preferences.GET_PREFERENCES)
```

Comment:
Why is this change needed? Won't this break stuff?

Reply:
https://github.com/fylein/fyle-qbo-api/blob/c4ee4dc2e252d92edbc6e9ad061c33f09c662aa6/apps/quickbooks_online/utils.py#L458

This will break no, we're not using ['Preferences'] key here